src/data.js

```diff
@@ -933,6 +933,30 @@ let Worker = Entity.configure({
 
     return item;
   },
+}).configure({
+  version:            5,
+  properties: {
+    provisionerId:    Entity.types.String,
+    workerType:       Entity.types.String,
+    workerGroup:      Entity.types.String,
+    workerId:         Entity.types.String,
+    /**
+     * List of objects with properties:
+     * - taskId
+     * - runId
+     * See JSON schema for documentation.
+     */
+    recentTasks:      Entity.types.JSON,
+    quarantineUntil:  Entity.types.Date,
+    // the time at which this worker should no longer be displayed
+    expires:          Entity.types.Date,
+    firstClaim:       Entity.types.Date,
+  },
+  migrate(item) {
+    item.quarantineUntil = new Date();
```

Comment:
I'm a little worried about clock skew here.  Could we default these to 1970 or something like that?

Reply:
Most definitely. `Date(0)` will set it to `1970-01-01T00:00:00.000Z`.